### DISCUSSION TITLE
dealing with ambient DNA in ATAC assay

### DISCUSSION THREAD

**Discussion by AmelZulji** [stuart-lab/signac#1300] [Q&A] [answered]
Dear Signac team, 

I suspect that my multiome-10x-genomics-ATAC samples are contaminated with ambient/backround DNA. Would you have any suggestions how to check background contamination and perhaps correct for it?

Thank you in advance and kind regards,
Amel

**Comment by timoast** [comment, marked as answer]
I don't know of any methods to explicitly account for ambient DNA in scATAC-seq analysis steps. However, it seems like this should be fairly uniform across cells that were processed together, and may be more of an issue if you have cells from different tissues that you want to compare across.

**Reply by AmelZulji** [reply]
Thank you for the answer, Tim! 

I am suspecting that the background signal - even though uniform across cells - is so high that it started masking cell-specific signal. So i was wondering whether is there any way to at least test it... However, as you said seems like there is nothing developed for that purpose. 

Kind regards,
Amel Zulji
